script.js

```diff
@@ -1,4 +1,145 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+
+
+// There are 2 game modes
+var gameModeDiceRoll = 'gameModeDiceRoll';
+var gameModeDiceOrder = 'gameModeDiceOrder';
```

Comment:
Good job storing your string game mode names in a variable! Decreases the chances of making a typo